kudo-operator/templates/spark-events-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Params.enableHistoryServer }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: spark-events-pvc
+  namespace: {{ .Namespace }}
+spec:
+  storageClassName: {{ .Params.storageClassName }}
```

Comment:
just wondering if we can omit always explicit configuration and rely on [DefaultStorageClass](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#defaultstorageclass) admission controller?

Reply:
I think it makes sense to use by default the `defaultStorageClass` but leave the door open for any other storage class using parameters 
https://github.com/kudobuilder/operators/blob/master/repository/kafka/operator/templates/statefulset.yaml#L157

Reply:
thanks for the pointer, @zmalik. With Spark, it turns out that `StorageClass` is too low-level abstraction because users require to mount volumes to every Spark Application they submit. So our thinking here is to provide instructions on how to create PVC and specify its name here instead of storage class and in volume mounts of each application.